src/UserFrontend.php

```diff
@@ -61,10 +64,26 @@ public static function login_head() {
    */
   public static function login_form_defaults(array $args) {
     $args['value_remember'] = TRUE;
+    $args['remember'] = FALSE;
     return $args;
   }
 
   /**
+   * @implements login_form
+   */
+  public static function login_form() {
+    ob_start(__CLASS__ . '::removesForgetmenotCheckbox');
```

Comment:
Hiding/removing the checkbox entirely from the markup is a good idea for the frontend (which is achieved with the simple parameter).

For the backend login form, I think the solution can be more simple.  I'd simply load/output CSS to hide the form element in the frontend.